web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -128,23 +135,9 @@ const PurchaseButton = styled.div`
   }
 `;
 
-const initalState = {
-  point: '0',
-  email: 'test@gmail.com',
-  password: 'asdf1234!',
-  name: 'jsontest',
-  phoneNumber: '010-1234-5678',
-  address: '서울특별시 성북구 정릉동 국민대학교 기숙사',
-  emailDisabled: false,
-  passwordDisabled: false,
-  nameDisabled: false,
-  phoneNumberDisabled: false,
-  addressDisabled: false,
-};
-
-const formatPhoneNumber = (value: string): string => {
+const changePhoneNumber = (value: string): string => {
   const phone = value.replace(/[^0-9]/g, '');
-  if (phone.length >= 10) {
+  if (phone.length >= 10 && phone.length < 13) {
```

Comment:
Do we need size of 12? Length of `01012345678` is 11. And the length of `0161234567` is 10.

Reply:
then we change phoneNumber minlength 11?

Reply:
Whatever is fine if the size is between 11 and 12.